### ISSUE TITLE
How does it identifies maintainers?

### ISSUE THREAD

**Issue by techtonik** [ansible/ansibullbot#1039]
I've got some `ansibullbot` questions:

1. Who is the maintainer?
2. Am I a maintainer?
3. Where is the list of maintainers for specific modules?

[Overview](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md#overview) could link to appropriate explanation.

**Comment by webknjaz** [comment]
It's identified from config: https://github.com/ansible/ansible/blob/devel/.github/BOTMETA.yml

**Comment by jctanner** [comment]
This is the entry point for fetching metadata about any given file, which also computes the maintainers...

https://github.com/ansible/ansibullbot/blob/master/ansibullbot/utils/component_tools.py#L995

**Comment by techtonik** [comment]
Just to make it absolutely clear - maintainers are not computed, but fetched from BOTMETA.yml. I mean that no source code history is processed, all processing is just matching filenames to modules and to maintainers.

**Comment by jctanner** [comment]
It is a "compute" operation because the authors of the file are also considered.

**Comment by techtonik** [comment]
Authors are from source code history?

**Comment by webknjaz** [comment]
@techtonik no, from module metadata history. There's a variable with YAML in it, in each module.

**Comment by techtonik** [comment]
Where is it? https://github.com/ansible/ansible/tree/devel/lib/ansible/modules/files

**Comment by techtonik** [comment]
I mean how is it called? https://github.com/ansible/ansible/blob/905acd7c9ebf151b19d3f48186ad758b93ec97a0/lib/ansible/modules/files/blockinfile.py#L11-L13

**Comment by webknjaz** [comment]
https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/files/file.py#L30

**Comment by webknjaz** [comment]
https://github.com/ansible/ansible/blob/905acd7c9ebf151b19d3f48186ad758b93ec97a0/lib/ansible/modules/files/blockinfile.py#L19-L20

**Comment by techtonik** [comment]
How can I check my status?
How can I list all maintainers for `blockinfile` including namespace maintainers and core members?

**Comment by webknjaz** [comment]
@techtonik you can only check botmeta config for entries where you're a maintainer + it takes into account module authors.
You can only use `bot_status` command in issues or PRs. I don't recall anything else we expose.

**Comment by mkrizek** [comment]
Re-opening this issue as a documentation request.
